CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+## 4.0.0
+⚠️ ⚠️ Important ⚠️ ⚠️ Observer mode for Amazon is not supported yet.
+
+### Features
+
+#### Amazon Appstore Support
+We have introduced support for using the Amazon Appstore. We have extensively tested this, and there are some apps using our pre-release Amazon versions in production.
+
+However, we have found some inconsistencies in the way Amazon Appstore prices are reported. We are actively working on patching these inconsistencies.
+
+Please help us help you by reporting any issues you find. [New RevenueCat Issue](https://github.com/RevenueCat/purchases-unity/issues/new/).
+
+You can enable Amazon Appstore support by configuring the SDK using the new `RevenueCatAPIKeyAmazon` field.
+
+For more information around configuration please take a look at the [Amazon Appstore section in our docs](https://docs.revenuecat.com/docs/amazon-platform-resources). The official [Amazon In-App Purchasing docs](https://developer.amazon.com/docs/in-app-purchasing/iap-overview.html) also contain very valuable information, especially around testing and best practices.
+
+#### StoreKit 2 support
+This version of the SDK automatically StoreKit 2 APIs under the hood only for APIs that the RevenueCat team has determined work better than StoreKit 1. 
+
+#### New types and cleaned up naming
+New types that wrap StoreKit's native types, and we cleaned up the naming of other types and methods for a more consistent experience. 
+
+### Removed APIs
+- `Identify` and `CreateAlias` have been removed in favor of `Reset`.
+- `GetEntitlements` has been removed in favor of `GetOfferings`.
+- `revenueCatAPIKey` has been removed in favor of `revenueCatAPIKeyApple`, `revenueCatAPIKeyGoogle` and `revenueCatAPIKeyAmazon`.
+
+### Renamed APIs
+
+| 3.x | 4.0.0 |
+| :-: | :-: |
+| `PurchaserInfo` | `CustomerInfo` |
+| `Transaction` | `StoreTransaction` |
+| `Product` | `StoreProduct` |
+| `PaymentDiscount` | `PromotionalOffer` |
+| `Purchases.RestoreTransactions` | `Purchases.RestorePurchases` |
+| `Purchases.GetPaymentDiscount` | `Purchases.GetPromotionalOffer` |
+| `Purchases.UpdatedPurchaserInfoListener` | `Purchases.UpdatedCustomerInfoListener` |
+| `Purchases.InvalidatePurchaserInfoCache` | `Purchases.InvalidateCustomerInfoCache`
```

Comment:
Should we mention all the lowercase to uppercase changes in the objects?

Reply:
added a few, basically all the ones for _existing_ objects. Not sure what to do with the ones that are for technically new objects, i.e.: `Product.foo` vs `StoreProduct.Foo`. Thoughts?